apache2/conf-enabled/sbf-custom-mime.conf

```diff
@@ -0,0 +1 @@
+../conf-available/sbf-custom-mime.conf
```

Comment:
Agreed.

So the reason this is in the base image is because it really has nothing to do with SBF. The file causes svgs to be compressed before they are served. The reason I prefixed it with 'sbf' was so if we ported to a newer version of apache we would know that file was a custom file we want to keep. Not sure if naming it like this is effective. I can change it to something generic.